en/getting-started/first-game/index.md

```diff
@@ -201,70 +512,62 @@ export class PlayerController extends Component {
+Then click the **Preview** button on the center of the top toolbar.
 
-__Now__, click the __Play__ button. Once running, click the left and right mouse buttons on the opened web page, you can see the following screen:
-
-![player move](./images/player-move.gif)
+![play button](./images/play.png)
 
-For additional details please refer to the [Project Preview Debugging](../../editor/preview/index.md) documentation.
+By clicking the left mouse button or right mouse button in the open browser, you may see the following screen:
 
-### Adding character animations
+![player move](./images/player-move.gif)
 
-The `Player` can be moved in a horizontal direction. This is a start, but not good enough. `Player` must become more life-like. This effect can be achieved by adding a vertical animation to the character.
+For more preview functions, please refer to [Project Preview & Debugging](../../editor/preview/index.md).
 
-> **Note**: before proceeding, please read the [Animation Editor](../../animation/index.md) documentation.
+## Add Character Animations
 
-After reading and understanding the capabilities of the __Animation Editor__ character animations can be implemented!
+We could notice that the horizontal movement of the character is tedious from the result of the above execution. To make out character feel like jumping you could add a vertical animation. For more information about **Animation Editor**, please refer to [Animation System](../../animation/index.md).
 
-1. Locate the __Animation__ panel, at the bottom of the editor alongside the __Assets Preview__ and the __Console__ panels. Select the `Body` node in the __Scene__ and click to add an __Animation Component__ and then click again tp create a new __Animation Clip__. Give this new __Animation Clip__ a name of `oneStep`.
+1. Select the `Body` node in the scene, and then click the **Add Animation Clip** button in the **Animation Editor** at the bottom of the editor to create an animation clip and name it `onStep`.
 
     ![player move](./images/add-animation.gif)
 
-2. Enter __animation editing mode__ in-order to add the __position attribute__. Next, add three __key frames__ with position values ​​of __(0, 0, 0)__, __(0, 0.5, 0)__, __(0, 0, 0)__.
+2. Enter the animation edit mode, add a `position` property track to the created clip, and then add three keyframes, the position value of each should be (0, 0, 0), (0, 0.5, 0), (0, 0, 0);
 
     ![add keyframe](./images/add-keyframe.gif)
 
-    > **Note**: remember to save the animation before exiting the animation editing mode, otherwise the animation will be lost.
+    > **Note**: Alway remember to save animation clips when exiting animation mode, otherwise you work will be lost.
```

Comment:
Alway -> always?
or  Don't forget to

when exiting animation mode -> when exiting animation editing mode